.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Substrate Connect CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-extension-versions:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - id: package-version
+      run: echo "::set-output name=version::`grep -rw './projects/extension/package.json' -e 'version' | sed 's/[^0-9.]//g'`"
```

Comment:
Is it possible to use `jq` instead maybe? It feels a bit cleaner to me.
Something like `grep package.json | jq .version` should work, I believe?